content/projects/remoting/index.adoc

```diff
@@ -69,6 +69,6 @@ There are several packages which bundle the Remoting library and allow connectin
 via one of remoting protocols.
 
 * link:https://hub.docker.com/r/jenkins/agent/[Docker Agent]: Base image, which bundles Remoting
-* link:https://hub.docker.com/r/jenkins/inbound-agent/[Docker Inbound Agent]: Image, which can be used to connect agents using TCP (JNLP protocols) or WebSockets
+* link:https://hub.docker.com/r/jenkins/inbound-agent/[Docker Inbound Agent]: Image, which can be used to connect agents using TCP or WebSocket
```

Comment:
Not worth mentioning JNLP here at all, since it is only used in deprecated code paths.